src/rest_api/routing.clj

```diff
@@ -55,6 +55,10 @@
     [route-spec]
     [route-spec opts]))
 
+(defn swagger-id-field [entity-type]
```

Comment:
perhaps use `entity-name` or maybe `entity-alias` here instead of `entity-type` (the "catalyst" name of the entity in the URL (e.g "disease" instead of "do-term" et al).